packages/components/src/Combobox/Combobox.types.ts

```diff
@@ -102,6 +107,7 @@ export interface ComboboxActivatorProps {
 export interface ComboboxTriggerProps
   extends Pick<ComboboxContentProps, "selected"> {
   readonly label?: string;
+  readonly activatorRef?: React.RefObject<HTMLButtonElement>;
```

Comment:
Minor comment, I think this should should React.Ref<HTMLButtonElement> because the ref might be a callback ref (this is what RHF uses). That way consumers don't need to cast it